### DISCUSSION TITLE
TimePicker警告

### DISCUSSION THREAD

**Discussion by veagxxx** [ant-design/ant-design#37103] [Q&A]
循环TimePicker出现会 **elements with non-unique id** 警告，如何去除

**Comment by zombieJ** [comment]
循环里配置了 id？元素 id 要不同，循环里要改 id。

**Reply by veagxxx** [reply]
警告的是TimePicker内部的input框重复id，没有id也算重复id，但是TimePicker并不提供id属性

**Reply by veagxxx** [reply]
![image](https://user-images.githubusercontent.com/33343134/185023720-583367b9-4a40-42f0-a0f5-29a28f82b74c.png)
